backend/database/transaction.go

```diff
@@ -117,6 +117,19 @@ func (tx *Transactable) Select(modelSlice interface{}, sb squirrel.SelectBuilder
 	return tx.sel(tx.tx.Select, modelSlice, sb)
 }
 
+// SelectRaw executes a raw SQL string
+// Note: this is for retriving multiple results, or "rows"
+func (tx *Transactable) SelectRaw(modelSlice interface{}, query string) error {
+	return tx.selRaw(tx.tx.Select, modelSlice, query)
+}
+
+// SelectRaw executes a raw SQL string with args
+// Note: this is for retriving multiple results, or "rows"
+func (tx *Transactable) SelectRawWithIntArg(modelSlice interface{}, query string, intVal int64) error {
```

Comment:
I think you should be able to remove this, and just add `, ...args interface{}` to `SelectRaw` to allow you to:
* pass in more than one thing
* pass in more than just an int

Reply:
ah yes! I looked into replacing the int with values []interface{}, but got an error at some point because to break apart the values in query, you'd have to use `pq.Array` like this `db.Query(`SELECT * FROM t WHERE id = ANY($1)`, pq.Array([]int{235, 401}))`. 

But since the actual db query call is abstracted behind a few functions in sqlx I would have to write a bunch of functions based on sqlx functions to get it to work (or I guess pass along an array wrapped in pq.Array (though I feel like the typing for that could get weird)). Although now that I think about it, I might be able to combine the sqlx functions into one custom function; I'll see if that's doable

Reply:
yeah this is pretty messy to implement (at least AFAI can tell) - I might look into it again if I need to pass additional args though, because it would be cleaner that having to implement a bunch of methods for specific type args